kafka-client/src/main/java/dev/responsive/kafka/internal/stores/CommitBuffer.java

```diff
@@ -498,11 +641,17 @@ private void throwFencedException(final RemoteWriteResult result, final long con
     );
     log.warn(msg);
 
+    flushFencedSensor.record();
```

Comment:
I double checked because I wasn't sure this was still right and indeed this code is no longer only called when we're fenced. any time the write fails, this code path will be triggered

we should probably update the entire error handling path here, but for now we can just rename the metric to reflect that it's any error in flush.

Reply:
talked offline, I'll push this change to the branch and merge the PR

Reply:
nice catch -- if anything it's probably more useful this way as we'd want to know if flushes fail whatever the reason may be.

If we realize that we do need a more fine-grained look then we can always go back and add additional metrics for the specific type of error, although generally I feel it's easy enough to check the logs and figure it out if/when we see a spike in the flush-errors metric